### ISSUE TITLE
Some not specified fields should not be updated to null

### ISSUE THREAD

**Issue by jeanpasquier75** [pyronear/pyro-api#103]
Currently when a PUT request is done and one optional field is not specified it is updated to null. 

A solution should be to add an option boolean to crud.put or crud.update_entry, if set to True it will update only the not-None fields

**Comment by florianriche** [comment]
I agree it's an issue. However I think we should alter the behavior by default and modify the fields only when they are specified.

**Comment by frgfm** [comment]
My bad, I think this issue isn't dealt with completely. I made some unittest and I can confirm it isn't resolved yet and here are the reasons I suspect:
1. some default values in the schema are not None (which is the check performed by #113)
2. the validator won't pass if None are in some fields

So we might need to find extra flexibility on the schema for UPDATE routes
